_posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "This week in Rails: Turbolinks 5, Rails 5.0.0.beta3 and more!"
+categories: news
+author: vipulnsward
+published: true
+date: 2016-03-04
+---
+
+####
+
+Hi All!
+
+This is [Vipul](https://twitter.com/vipulnsward) bringing you the latest news from [Rails](https://github.com/rails/rails)!
+
+## Featured
+
+### [This week's Rails contributors](http://contributors.rubyonrails.org/contributors/in-time-window/20160227-20160304)
+
+This week 48 contributors helped to move Rails forward! Want to be one of them? Look at the [issues](https://github.com/rails/rails/issues) list and make a contribution!
+
+### [Rails 4.2.6.rc1 and 4.1.15.rc1 have been released](http://weblog.rubyonrails.org/2016/3/1/Rails-4-2-6-rc1-and-4-1-15-rc1-have-been-released/)
+
+Rails 4.2.6.rc1 and 4.1.15.rc1 have been released!
+If no regressions are found expect the final release around March 4, 2016.
+
+### [Publish AS::Executor and AS::Reloader APIs](https://github.com/rails/rails/pull/23807)
+
+These both allow external code to run blocks of user code to do "work" at a similar unit size to a web request without needing to get intimate with `ActionDispatch`.
+
+This change is intended to allow interested callers (`Sidekiq`, `ActionCable`), to just do:
+
+    Rails.application.reloader.wrap do
+      # run some user code
+    end
+
+and Rails will take care of the interlock, code reloading, returning `ActiveRecord` connections to the pool, and anything else that might be relevant.
```

Comment:
@fxn I believe you mean't all mentions of components here need to be changed. `ActiveRecord` => Active Record.
I pulled this mostly from https://github.com/rails/rails/pull/23807